assets/css/layout/_footer.scss

```diff
@@ -0,0 +1,158 @@
+// Footer styles.
+
+footer {
+  background: $navy-light;
+  color: $white;
+  margin-top: rem(30);
+  padding: rem(16) rem(32);
+
+  .footer-content {
```

Comment:
This needs `margin-left: auto` and `margin-right: auto` for large screens:

<img width="2548" alt="Screen Shot 2020-03-30 at 10 18 28 AM" src="https://user-images.githubusercontent.com/605361/77936244-e066ef00-726f-11ea-98be-a0ae1d9a2251.png">